Hardcoded module names

Just got everything working and am now getting errors that it can't find a couple of modules/files during my tests:

Cannot find module 'testing/api-http-service.stub' from 'user.service.spec.ts'
Cannot find module 'testing/user-service.stub' from 'register-form.component.spec.ts'

I've got a folder inside `/src` called `testing` and import stubs for my tests like:
```
import { RouterStub } from 'testing/router.stub';
```

Was trying to figure out why and I see jest-preset only whitelists app and environments like:
```
"moduleNameMapper": {
  "app/(.*)": "<rootDir>/src/app/$1",
  "environments/(.*)": "<rootDir>/src/environments/$1"
},
```

Angular CLI allows anything inside `src`

--- a/jest-preset.json
+++ b/jest-preset.json
@@ -13,8 +13,7 @@
     "html"
   ],
   "moduleNameMapper": {
-    "app/(.*)": "<rootDir>/src/app/$1",
-    "environments/(.*)": "<rootDir>/src/environments/$1"
+    "(.*)": "<rootDir>/src/$1"
   },
   "transformIgnorePatterns": [
     "node_modules/(?!@ngrx)"

